scripts/dnd5e-helpers.js

```diff
@@ -906,11 +946,38 @@ function ReactionDetect_preCreateChatMessage(msg){
     /** hand over to reaction apply logic (checks combat state, etc) */
     ReactionApply(speaker.actor, speaker.token, itemId);
   }
-}
+};
 
 Hooks.on("preCreateChatMessage", async (msg, options, userId) => {
  const reactMode = game.settings.get('dnd5e-helpers', 'cbtReactionEnable');
   if ( reactMode === 1 || reactMode === 3) {
     ReactionDetect_preCreateChatMessage(msg);
   }
+
+  let deathSaveRoll = msg.flavour.match(/Death Saving Throw/)
+  let rollTotal = parseInt(msg.content)
+if(rollTotal <= 5 && deathSaveRoll && (game.settings.get('dnd5e-helpers', 'owDeathSave'))){
+  let actor = game.actors.get(msg.speaker.actor)
+  OpenWounds(actor)
+}
+
+let rollResult = msg.roll.match(/"result":[0-9]{1,2}/)
```

Comment:
Probably a better idea to turn the 'roll' string into a proper JSON object for type safety.
```suggestion
  const rollJson = getProperty(msg, "roll");
  if (rollJson){
    const rollData = JSON.parse(rollJson);
    const rollResult = rollData.result; // <- not sure of the exact field you are regex searching for here, modify as needed.
  }
```

Reply:
the field is just searching for the "result": (dice value here) part of the rollJson. This returns the exact dice value of the roll rather than the result. This is needed to search for critital hits